packages/x-tailwindcss/demo/src/components/xds-layout-single-column.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <XdsBaseShowcase #default="{ cssClass }" title="Layout single column" :sections="sections">
+    <button @click="openModal(cssClass)" class="x-button">Open layout example</button>
+
+    <dialog :ref="cssClass" class="modal x-bg-neutral-10">
+      <div :class="[cssClass, 'x-layout-min-margin-48']">
+        <div class="x-layout-item x-bg-neutral-0 x-border-b-1 x-border-neutral-25">
+          <div class="x-flex x-justify-between x-items-center x-py-8">
+            <span class="x-title3">SINGLE COLUMN LAYOUT</span>
+            <form method="dialog" class="x-flex">
+              <button @click="enableScroll" class="x-button x-button-ghost" value="default">
+                Close
+              </button>
+            </form>
+          </div>
+        </div>
+
+        <div class="x-layout-item x-border-b-1 x-border-neutral-25">
+          <div class="x-flex x-justify-between x-items-center x-py-16">
+            <div class="x-title3">HEADER START</div>
+            <div class="x-title2">HEADER MIDDLE</div>
+            <div class="x-title3">HEADER END</div>
+          </div>
+        </div>
+
+        <div class="x-layout-item x-border-b-1 x-border-neutral-25">
+          <div class="x-flex x-justify-center x-p-16 x-title3">SUB HEADER</div>
+        </div>
+
+        <div class="x-layout-item x-layout-expand x-layout-no-margin-right x-bg-neutral-25">
```

Comment:
I think here we can just add the `x-scroll` class to the item:

```
        <div
          class="x-layout-item x-scroll x-layout-expand x-layout-no-margin-right x-bg-neutral-25"
        >
          <div class="x-flex x-flex-col x-justify-between x-h-[2000px] x-p-12">
            <section>MAIN</section>
            <span class="x-place-self-end">MAIN BOTTOM</span>
          </div>
        </div>
```

Reply:
Good point! Finally, I have refactored it a bit.